package.json

```diff
@@ -22,17 +22,17 @@
   "homepage": "https://github.com/btholt/sfo#readme",
   "devDependencies": {},
   "dependencies": {
-    "babel-core": "^6.26.0",
+    "@babel/core": "^7.0.0-beta.35",
     "babel-eslint": "^8.0.1",
-    "babel-loader": "^7.0.0",
-    "babel-plugin-syntax-dynamic-import": "^6.18.0",
-    "babel-plugin-syntax-jsx": "^6.18.0",
-    "babel-plugin-transform-class-properties": "^6.24.1",
-    "babel-plugin-transform-es2015-modules-commonjs": "^6.24.1",
-    "babel-plugin-transform-react-jsx": "^6.24.1",
-    "babel-preset-env": "^1.5.1",
-    "babel-preset-flow": "^6.23.0",
-    "babel-runtime": "^6.26.0",
+    "babel-loader": "^8.0.0-beta.0",
```

Comment:
Don't worry, this `babel-loader` only add support for `@babel/core` no new features were added.